### PR TITLE
Updates EE dialog styles and content

### LINKED ISSUE #3392: Formatting inside modal could use some work.
## Description
The formatting on the Diversity and Inclusion modals seems off. @JoshBeveridge will take a look

## Screenshot: Prototype
image

### PR DESCRIPTION
**Pull request by JoshBeveridge**
# Requirements
- [x] update dialog styles according to #3392

# Notes
Resolves #3392

### CODE REVIEW

File: frontend/talentsearch/src/js/components/employmentEquityForm/dialogs/DisabilityDialog.tsx [hunk 1]
```diff
@@ -68,25 +68,27 @@ const DisabilityDialog: React.FC<EquityDialogProps> = ({
     <Dialog
       isOpen={isOpen}
       onDismiss={onDismiss}
-      color="ts-primary"
+      color="ts-secondary"
       title={intl.formatMessage(getEmploymentEquityGroup("disability"))}
     >
       <UnderReview />
-      <p data-h2-margin="base(0, 0, x1, 0)">
+      <div data-h2-margin="base(x1, 0)">
```
Comment (esizer) [inline frontend/talentsearch/src/js/components/employmentEquityForm/dialogs/DisabilityDialog.tsx:75]: We could just add `data-h2-margin="base(x1, 0)"` to the `<p>` in `Definition.tsx` so we no longer need the wrapping `<div>` in each `Dialog`

**Review by esizer — APPROVED** [review]
Looks good, added a small suggestion but won't let that hold this up.

File: frontend/talentsearch/src/js/components/employmentEquityForm/dialogs/DisabilityDialog.tsx (hunk 1 above)
Comment (JoshBeveridge) [reply frontend/talentsearch/src/js/components/employmentEquityForm/dialogs/DisabilityDialog.tsx:75]: Oh, that's much cleaner! I'll rework it and resubmit the review.

Comment (JoshBeveridge) [reply frontend/talentsearch/src/js/components/employmentEquityForm/dialogs/DisabilityDialog.tsx]: Sorry, I forgot you guys were trying to deploy this to Fable today!

**Review by esizer — APPROVED** [review]
Looking great! 😁

**Pull request merged**